scripts/gis-material-update/README.md

```diff
@@ -323,6 +324,43 @@ Output files (names configured in `config.yaml`)
 
 - tormays_central_business_areas.gpkg
 
+### `liikennevaylat`
+
+Prerequisite: 
+- `central_business_area` material fetched
+- `ylre_katuosat` material fetched
+- `liikennevaylat` material fetched
+- `central_business_area` material processed
+- `ylre_katuosat` material processed
+
+In gis-material-update/.env file there need to be following secrets:
+
+```sh
+HELSINKI_EXTRANET_USERNAME=
+HELSINKI_EXTRANET_PASSWORD=
+```
+These secrets can be found from Azure Key vault or from OpenShift secrets (haitaton-secret).
+
+Docker example run (ensure that image build and file copying is
+already performed as instructed above):
+
+```sh
+docker-compose up -d gis-db
+docker-compose run --rm gis-fetch liikennevaylat central_business_area ylre_katuosat
+docker-compose run --rm gis-process central_business_area ylre_katuosat
+docker-compose run --rm gis-process liikennevaylat
+docker-compose stop gis-db
+```
```

Comment:
It is a bit confusing trying to follow these instructions because these fetch and process for central_business_area and ylre_katuosat are already before this in their own sections. Maybe we don't need "Run data fetch" section at all or maybe it should be under this "Run processing" section? Or maybe some instruction in top of the README.md could say that "In order to run both data fetch and processing, see "Run processing""?